tools/test.py

```diff
@@ -1642,6 +1645,17 @@ def Main():
   all_unused = [ ]
   unclassified_tests = [ ]
   globally_unused_rules = None
+  if options.type is not None:
+    env_type = options.type
+  else:
+    if "fips" in subprocess.check_output(["node", "-p",
```

Comment:
can we assume `node` has been installed or path has been appropriately set so that this just works?

Reply:
Not necessarily. The script uses `context.GetVM()` to locate the executable.

Reply:
very helpful. thanks